sdk/core/Azure.Core.Experimental/src/DynamicData/DynamicCaseMapping.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Dynamic
+{
+    /// <summary>
+    /// Options for getting and setting DynamicData properties.
```

Comment:
If we are planning to keep indexer property access as direct pass-through no matter which option is set here, we should document that explicitly here and on the options type.

Reply:
Cool, I'll update that.